src/pudl/metadata/resources/vceregen.py

```diff
@@ -0,0 +1,37 @@
+"""Table definitions for data coming from the Vibrant Clean Energy Renewable Generation Profiles."""
+
+from typing import Any
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "out_vceregen__hourly_available_capacity_factor": {
+        "description": (
+            "The data in this table were produced by Vibrant Clean Energy, and are "
+            "licensed to the public under the Creative Commons Attribution 4.0 International "
+            "license (CC-BY-4.0). The table consists of hourly, county-level renewable "
+            "generation profiles in the continental United States and are compiled based on "
+            "outputs from the NOAA HRRR weather model. Profiles are stated as a capacity "
+            "factor (a fraction of nameplate capacity) and exist for onshore wind, offshore "
+            "wind, and fixed-tilt solar generation types."
+        ),
+        "schema": {
+            "fields": [
+                "hour_utc",
+                "hour_of_year",
+                "report_year",
+                "county_id_fips",
+                "county",
+                "state",
+                "latitude",
+                "longitude",
+                "capacity_factor_solar_pv",
+                "capacity_factor_onshore_wind",
+                "capacity_factor_offshore_wind",
+            ],
+            "primary_key": ["hour_utc", "county_id_fips", "county"],
```

Comment:
This is kind of weird and annoying, but the FIPS table provided by VCE has multiple county names per FIPS code. I'm not exactly sure why? Maybe from multiple years? This makes it so that hour and county_id alone aren't unique identifiers for the data.

Reply:
Hmm. This seems like something we might want to ask Chris about, since it suggests that the "counties" might not actually be counties as described in the README. If it really is the case that there are multiple different locations with data being reported that are part of the same FIPS code, then maybe we have no choice but to use the county as the primary key, and NOT the FIPS code, since it then becomes dependent on the county/place name. But this will mess with any attempt to connect the data to geospatial information / the county geometries.

Reply:
Yeah, it's tricky. I think for now I'm going to make this `county` column a `region` or something similar column. We need either `county_id_fips` or `state` as part of the primary key to differentiate between counties of the same name.

Reply:
If these non-county things are lakes or city centers like Ella suggested then I'm curious if they're really supposed to be in there, or what geometry they're supposed to be linked to -- all of these capacity values are averages of the gridded model outputs from some area.

Reply:
We should ask Chris, I'll get a list of them.

Reply:
Unclear, but it seems like Chis did know they were there. He said we could average the counties and cities so I did that. Left the lakes in.